tests/src/Functional/EntityOperationsTest.php

```diff
@@ -16,12 +16,12 @@ class EntityOperationsTest extends GroupEntityOperationsTest {
   /**
    * {@inheritdoc}
    */
-  public static $modules = ['group_webform'];
+  protected static $modules = ['group_webform'];
 
   /**
    * {@inheritdoc}
    */
-  public function provideEntityOperationScenarios() {
+  public function provideEntityOperationScenarios(): array {
```

Comment:
I rather wish I hadn't looked at this.

So the module doesn't test anything useful. Its basically unfinished.

However it will provide the relation for a webform (config) entity to a group still as that code hasn't changed.